_components/summary-box/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/summary-box/accessibility-tests/
+layout: accessibility-test
+component:
+ name: Summary box
+title: Summary box accessibility tests
+category: Components
+lead: Any USWDS Summary box component should pass these manual accessibility tests.
```

Comment:
```suggestion
lead: Any USWDS summary box component should pass these manual accessibility tests.
```

Reply:
Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)